src/main/java/com/github/alex1304/jdash/exception/BadResponseException.java

```diff
@@ -11,11 +13,15 @@
 	private final HttpClientResponse response;
 	
 	public BadResponseException(HttpClientResponse response) {
-		super(response.status().toString());
-		this.response = response;
+		this.response = Objects.requireNonNull(response);
```

Comment:
why is this change needed ? The default `toString()` of `Throwable` is just fine imo

Reply:
Reverted 5888910

Reply:
you forgot to remove the toString() override you added

Reply:
oof my mistake

deleted 2f1776c